source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -251,7 +252,7 @@ following commands:
    sudo cp /usr/share/defaults/crio/crio.conf /etc/crio/
    sudo $EDITOR /etc/crio/crio.conf
 
-Learn more about `Stateless in Clear Linux`_ in the `Clear Linux documentation`_.
+Learn more about `Stateless in Clear Linux`_ and view the `Clear Linux documentation`_.
 
 Proxy configuration (optional)
 ******************************
```

Comment:
line 263: use of all caps for emphasis. Should instead use italics e.g. *explicitly included*

line 309: "verify what are the proxy" should be "verify the proxy"

line 313: "permanently and available from" should be "permanently, and make them available for"